libs/template/themes/design-tokens/src/mobile-backoffice/index.ts

```diff
@@ -22,30 +20,19 @@ export const backofficeNgTokens: DesignToken[] = [
       typographySmallTokens.typography
     ),
     ...layoutTokens,
-    layout: {
-      container: {
-        width: '414px',
-        bleed: '10px',
-      },
+    container: {
+      width: '414px',
+      bleed: '0px',
     },
-  },
-  {
-    media: {
-      screen: Size.Lg,
-    },
-    ...layoutTokens,
-  },
-  {
-    media: {
-      screen: Size.Md,
+    modal: {
+      width: '404px',
+      bleed: '5px',
```

Comment:
I was thinking to do this inside the modal component, so that any layout wil benefit from this, regardless of the breakpoint / container. It means that the max-width of the modal will be limited to the container size (minus bleed).